Assets/Scripts/BeeController.cs

```diff
@@ -31,15 +40,14 @@ void Update()
     void GotoNextPoint()
     {
         // Returns if no points have been set up
-        if (points.Length == 0)
+        if (navObjects.Count == 0)
             return;
 
         // Set the agent to go to the currently selected destination.
-        agent.destination = points[destPoint].position;
+        agent.destination = navObjects[destPoint].position;
 
-        // Choose the next point in the array as the destination,
-        // cycling to the start if necessary.
-        destPoint = (destPoint + 1) % points.Length;
+
+        destPoint = (destPoint + 1);
```

Comment:
What happens, when the destPoint is bigger than the navObject count?

Reply:
Errors are thrown, a lot of them 👍

Reply:
than maybe we could restrict that? 😄